### PR TITLE
Add Fast-PT installation guide to docs

### PR DESCRIPTION
**Pull request by tilmantroester**
Added notes on how to install `Fast-PT` to docs, as well as the error message when `Fast-PT` is missing.

### CODE REVIEW

**Comment by tilmantroester** [comment]
I accidentally branched this one off #748. What's the best practice here? Should I just leave it as is, make a new PR, revert the commits, or rebase & `git push --force`?

**Comment by jablazek** [comment]
I think that in the docs, calling the package `FAST-PT` is best. This is consistent with what we call it in papers and the github repo, and should reduce errors in people pip installing the wrong package.

**Comment by tilmantroester** [comment]
I think I called it Fast-PT everywhere. Did I write it somewhere without the hyphen?

File: readthedocs/source/developer_installation.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ To install CCL using a ``pip`` developer installation, you can execute
 from the top-level directory in the repository. You will need ``CMake`` in
 order to install CCL in this way. See :ref:`getting-cmake` for help installing
 ``CMake`` if you do not already have it. In order to run the tests,
-you will need both ``CAMB`` and ``CLASS`` installed. See the instructions for
-:ref:`boltzmann-codes` for details.
+you will need ``CAMB``, ``CLASS``, and ``FastPT`` installed. See the instructions for
```
Comment (jablazek) [inline readthedocs/source/developer_installation.rst]: e.g. here. In general, it's best to have everything capitalized: FAST-PT

File: readthedocs/source/installation.rst [hunk 2]
```diff
@@ -70,6 +70,19 @@ An installation with ``pip`` should work as well. See the `CAMB <https://github.
 repo for more details. Note that if you installed CCL with ``conda``, ``camb``
 should already be in your environment.
 
+.. _getting-fast-pt:
+
+Getting FastPT
+==============
+
+To use ``FastPT`` with CCL, you can install it with:
```
Comment (jablazek) [inline readthedocs/source/installation.rst]: and here.

File: readthedocs/source/installation.rst [hunk 3]
```diff
@@ -70,6 +70,19 @@ An installation with ``pip`` should work as well. See the `CAMB <https://github.
 repo for more details. Note that if you installed CCL with ``conda``, ``camb``
 should already be in your environment.
 
+.. _getting-fast-pt:
+
+Getting FastPT
```
Comment (jablazek) [inline readthedocs/source/installation.rst]: here

File: readthedocs/source/developer_installation.rst (hunk 1 above)
Comment (tilmantroester) [reply readthedocs/source/developer_installation.rst]: Capitalised FAST-PT now.

File: readthedocs/source/installation.rst (hunk 2 above)
Comment (tilmantroester) [reply readthedocs/source/installation.rst]: Done. I think you were also looking at an old version without the hyphens.

File: readthedocs/source/installation.rst (hunk 3 above)
Comment (tilmantroester) [reply readthedocs/source/installation.rst]: Done.

**Comment by jablazek** [comment]
ah, sorry if I was looking at an old version.

**Comment by elisachisari** [comment]
Unfortunately, tests are still failing for me (see #735). 
[log.txt](https://github.com/LSSTDESC/CCL/files/4415557/log.txt)

**Comment by tilmantroester** [comment]
This PR doesn't touch any code, so this shouldn't affect any tests. Note that I did a force push to remove some commits, so you might need to get a clean copy of the branch.

**Comment by jablazek** [comment]
> Unfortunately, tests are still failing for me (see #735).
> [log.txt](https://github.com/LSSTDESC/CCL/files/4415557/log.txt)

this PR shouldn't have anything to do with the failures, right? I think you mean https://github.com/LSSTDESC/CCL/pull/746

File: readthedocs/source/installation.rst [hunk 4]
```diff
@@ -5,7 +5,7 @@ Installation
 CCL can be installed from ``pip``, ``conda``, or directly from source.
 It is configured to install most of its requirements automatically. However, if
 you want to use CCL with Boltzmann codes like ``CLASS`` or ``CAMB``, or do 
-perturbation theory calculations with ``Fast-PT``, you will
+perturbation theory calculations with ``FAST-PT``, you will
 need to make sure the ``Python`` wrappers for these packages are installed
```
Comment (jablazek) [inline readthedocs/source/installation.rst:9]: code and ``Python`` wrappers...

**Comment by jablazek** [comment]
I made one minor comment. Otherwise, looks good to me. Thanks again @tilmantroester!

**Comment by c-d-leonard** [comment]
Looks like we're ready to merge this, I'm going to hit the green button.

**Comment by c-d-leonard** [comment]
@tilmantroester are you okay with deleting the branch? we can get it back if we need to do, just like to keep the repo clean once the PR is merged. If you're okay with it go ahead and delete?

**Pull request merged**